go/consensus/tendermint/tendermint.go

```diff
@@ -1338,6 +1350,10 @@ func init() {
 	Flags.StringSlice(CfgP2PPersistentPeer, []string{}, "Tendermint persistent peer(s) of the form ID@ip:port")
 	Flags.Bool(CfgP2PDisablePeerExchange, false, "Disable Tendermint's peer-exchange reactor")
 	Flags.Bool(CfgP2PSeedMode, false, "run the tendermint node in seed mode")
+	Flags.Int(CfgP2PMaxNumInboundPeers, 40, "Max number of inbound peers")
+	Flags.Int(CfgP2PMaxNumOutboundPeers, 10, "Max number of outbound peers (excluding persistent peers)")
```

Comment:
Based on feedback we should increase this default as 10 seems low?

Reply:
Increased default to 20